topics/single-cell/tutorials/bulk-deconvolution-evaluate/tutorial.md

```diff
@@ -0,0 +1,585 @@
+
+> <hands-on-title>Plot the actual and inferred data</hands-on-title>
+>
+> 1. {% tool [Scatterplot with ggplot2](toolshed.g2.bx.psu.edu/repos/iuc/ggplot2_point/ggplot2_point/3.4.0+galaxy1) %} with the following parameters:
+>    - {% icon param-file %} *"Input in tabular format"*: `Results Table (Music)`
+>    - *"Column to plot on x-axis"*: `2`
+>    - *"Column to plot on y-axis"*: `3`
+>    - *"Plot title"*: `Correlation between inferred and actual cell-type proportions`
+>    - *"Label for x axis"*: `Actual proportions`
+>    - *"Label for y axis"*: `Inferred proportions`
+>    - In *"Advanced options"*:
+>       - *"Plotting multiple groups"*: `Plot multiple groups of data on one plot`
+>           - *"column differentiating the different groups"*: `1`
+>           - *"Color schemes to differentiate your groups"*: `Paired - predefined color pallete (discrete, max=12 colors)`
+>           - *"Reverse color scheme"*: `Default order of color scheme`
+>    - In *"Output options"*:
+>       - *"width of output"*: `6.0`
+>       - *"height of output"*: `4.0`
+>
+> 2. **Rename** {% icon galaxy-pencil %} output `Scatter plot - Music`
+>
+> 3. Add a `#plot` tag to `Scatter plot - Music`
+>
+> 4. {% tool [Scatterplot with ggplot2](toolshed.g2.bx.psu.edu/repos/iuc/ggplot2_point/ggplot2_point/3.4.0+galaxy1) %} with the following parameters:
+>    - {% icon param-file %} *"Input in tabular format"*: `Results Table (NNLS)`
+>    - *"Column to plot on x-axis"*: `2`
+>    - *"Column to plot on y-axis"*: `3`
+>    - *"Plot title"*: `Correlation between inferred and actual cell-type proportions`
+>    - *"Label for x axis"*: `Actual proportions`
+>    - *"Label for y axis"*: `Inferred proportions`
+>    - In *"Advanced options"*:
+>       - *"Plotting multiple groups"*: `Plot multiple groups of data on one plot`
+>           - *"column differentiating the different groups"*: `1`
+>           - *"Color schemes to differentiate your groups"*: `Paired - predefined color pallete (discrete, max=12 colors)`
+>           - *"Reverse color scheme"*: `Default order of color scheme`
+>    - In *"Output options"*:
+>       - *"width of output"*: `6.0`
+>       - *"height of output"*: `4.0`
+>
+> 5. **Rename** {% icon galaxy-pencil %} output `Scatter plot - NNLS`
+>
+> 6. Add a `#plot` tag to `Scatter plot - NNLS`
+>
+{: .hands_on}
+
+The output of this tool should produce two scatter plots that looks like the image below. Each point on the plot represents a cell-type for a specific subsample, so there should be 20 points of each colour (one for each subsample created earlier). Since we are comparing the actual and inferred proportions, the ideal scatter plot would have all of the points be at the `y=x` line. The further the deviations are from this ideal line, the less accurate the tool is. We can also use this plot to determine if the tool is under or over predicting proportion values for each cell-type, or if the tool is struggling to predict certain cell types.

```

Comment:
I think it should be noted that a scatter plot in which all points lie along along the y=x line would be the result of a perfect recapitulation of the single-cell proportions, but that single-cell atlases do not exactly reflect the in vivo cell-type proportion.